packages/beacon-node/src/chain/prepareNextSlot.ts

```diff
@@ -156,6 +157,17 @@ export class PrepareNextSlotScheduler {
             feeRecipient,
           });
         }
+
+        // If emitPayloadAttributes is true emit a SSE payloadAttributes event
+        if (this.chain.opts.emitPayloadAttributes === true) {
+          const data = await getPayloadAttributesForSSE(fork as ForkExecution, this.chain, {
+            prepareState,
+            prepareSlot,
+            parentBlockRoot: fromHex(headRoot),
+            feeRecipient: "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
why is this always `ZERO_HASH`?

Reply:
hmmm it could be set to the proposer's fee recipient  (or the beacon's default in case we don't have one), it any way seems to be irrelevant/ignorable  

![image](https://user-images.githubusercontent.com/76567250/226531317-423e8aa3-21ed-454e-9921-1d9898911ccc.png)

https://github.com/ethereum/beacon-APIs/pull/305/files#diff-3cc6eaa800c1a4bd6adcd78f1722b466f7a24048a66e320c8f92872fbcd9eefbR120